src/lib/formatDateClaim.js

```diff
@@ -0,0 +1,22 @@
+import moment from "moment";
+export default function formatDateClaim(claim) {
```

Comment:
I already wrote a function for it using moment also supporting BC dates, I can merge it

Reply:
Curious to see how you convert julian dates  😁

Reply:
oh I don't think I did that

Reply:
hehe, do you want to have a look to a conversion function for that? pretty sure moment does not support it, otherwise for the sake of simplicity we could show the date as is adding an icon if it's in julian